packages/react-dom/src/__tests__/ReactNestedUpdateBug-test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ * @jest-environment node
+ */
+
+// This is a regression test for https://github.com/facebook/react/issues/13188.
+// It reproduces a combination of conditions that led to a problem.
+
+if (global.window) {
+  throw new Error('This test must run in a Node environment.');
+}
+
+// The issue only reproduced when React was loaded before JSDOM.
+const React = require('react');
+const ReactDOM = require('react-dom');
+
+// Unlike other tests, we want to enable error logging.
+// Note that we'll later override console.error.
+Error.prototype.suppressReactErrorLogging = false;
```

Comment:
For other onlookers, I thought this meant that React extended the Error prototype. This only happens in the React test suite:
https://github.com/facebook/react/blob/f762b3abb1d951b366f3c8a2354aeedb7fbc32ff/scripts/jest/setupEnvironment.js#L10-L16

Reply:
Actually looks like this raises a warning on CI:

>  24:1  warning  Error prototype is read only, properties should not be added  no-extend-native

Can you add: `// eslint-disable-next-line no-extend-native`?